web/src/main/java/se/inera/intyg/webcert/web/service/monitoring/MonitoringLogServiceImpl.java

```diff
@@ -241,57 +519,167 @@ public void logPrivatePractitionerTermsApproved(String userId, Personnummer pers
 
     @Override
     public void logNotificationSent(String hanType, String unitId, String intygsId) {
-        logEvent(MonitoringEvent.NOTIFICATION_SENT, hanType, unitId, intygsId);
+        try (MdcCloseableMap ignored = MdcCloseableMap.builder()
```

Comment:
This comment is for logPrivatePractitionerTermsApproved

Should we add some Mdc fields for the log?

Suggest that we add the following.

user.id: userId
user.role: AuthoritiesConstants.ROLE_PRIVATLAKARE

Reply:
Hmm did I miss this one... fixed